app/views/posts/_post.html.erb

```diff
@@ -71,14 +71,14 @@
     <div class="modal-content">
       <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title">Flag Post for Review</h4>
+        <h4 class="modal-title"><%= t(:flag_post_for_review) %></h4>
```

Comment:
We generally provide a `default` for each of the `i18n` strings that is equal to what is provided in the `en.yml` file.